dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.11-slim
 
 RUN apt update && apt upgrade -y
-COPY hpath-sim/requirements.txt /requirements.txt
+COPY hpath-sim/requirements.in /requirements.txt
```

Comment:
I think the existing method is fine -- use "pip-compile" to generate the requirements.txt file. (See the existing update-deps.sh file in yinchi-dev; you may need to pip-install pip-tools first.)